queries.sql

```diff
@@ -10,3 +10,53 @@ SELECT name, escape_attempts from animals WHERE weight_kg > 10.5;
 SELECT * from animals WHERE neutered = true;
 SELECT * from animals WHERE name != 'Gabumon';
 SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+
+/* transactions */
+/* Transaction1 */
+BEGIN;
+Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
```

Comment:
- [ ] You did well writing this transaction query. However, it is recommended that you verify that the change was made and rollbacked. 

Please see example:
```sql
   BEGIN; -- start transaction

   UPDATE animals
   SET species = 'unspecified'; -- make change

   SELECT species from animals; -- verify that change was made
   ROLLBACK;

   SELECT species from animals; -- verify that change was undone
```

Reply:
Thanks for acknowledging, in every transaction I have tried to track change with "Select * from animals", but not with specific coloumn. In screenshots it is visible. Thanks once again, I will update the queries.sql file for future reference.